collection-scripts/gather_sos_networking

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="/must-gather"
+LOG_PATH="${BASE_COLLECTION_PATH}/sos_info/networking"
+
+# use quorum-guard becuase it is privileged and uses hostNetwork
+QUORUM_GUARD_DATA=($(oc get pods -n openshift-etcd -l name=etcd-quorum-guard -o jsonpath='{range .items[*]}{.metadata.name}{","}{.status.hostIP}{" "}{end}')) || {
+  echo "ERROR: No networking data was collected"
+  exit 1
```

Comment:
will this affect the return code of the entire command?  I don't think we want to fail like this from a gathering tool

Reply:
yeah if oc has error we exit 1 I can tolerate oc failure.

Reply:
now exits 0 I need to test this within context of the tool to understand failure better.